homeassistant/components/light/abode.py

```diff
@@ -68,7 +70,9 @@ def is_on(self):
     def brightness(self):
         """Return the brightness of the light."""
         if self._device.is_dimmable and self._device.has_brightness:
-            return self._device.brightness
+            # Convert Abode brightness (0-100) to HASS brightness (0-255)
+            # Abode will return 100 during initialization of Abode
+            return ceil(int(self._device.brightness) * 255 / 100)
```

Comment:
`99 * 255 / 100 = 252.45`. It should be 255! What about `brightness * 255 / 99`

Reply:
Just realized your comment. Hmmm...

Reply:
Yeah I had it set to divide by 99 but noticed after restarting HA, Abode was returning a brightness value of 100 from one of my dimmers that was on which caused it to convert to 257. When you set the brightness to 100 is when it will only return 99.

Reply:
In this case I would remap the brightness of 100:

```
brightness = int(self._device.brightness)
if brightness == 100:
  return 255
else
  return ceil(brightness * 255 / 99)
```